src/main/java/com/google/jenkins/plugins/computeengine/ui/helpers/PreemptibleVm.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 CloudBees, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jenkins.plugins.computeengine.ui.helpers;
```

Comment:
Suggest `com.google.jenkins.plugins.computeengine.config` for these classes, since they are used also by JCasC, so not exactly limited to the “UI”.

Reply:
makes sense, thanks for the suggestion